.github/workflows/test-docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Test in docker
+
+on:
+  push:
+
+jobs:
+  test:
+    runs-on: ['ubuntu-22.04']
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tarantool:
+          - '1.10'
+          - '2.10'
+          - '2.11'
+
+        image:
+          - ubuntu:focal
+          - ubuntu:jammy
+          - debian:buster
+          - debian:bullseye
+
+        # Only 2.11 is available on debian bookworm
+        include:
+          - tarantool: '2.11'
+            image: debian:bookworm
+
+    container:
+      image: ${{ matrix.image }}
+
+    env:
+      TARANTOOL_CACHE_KEY_SUFFIX: -DA-${{ github.run_id }}
```

Comment:
Shouldn't we include `${{ matrix.image }}` into the cache key? It seems that `${{ github.run_id }}` is shared across jobs generated from the same job matrix.

https://github.com/tarantool/setup-tarantool/blob/615e8f4e4dfbf4e41631a72660d27f1ede102aa0/.github/workflows/test.yml#L334-L337

Reply:
Thanks! Fixed.